draft-patton-cfrg-vdaf.md

```diff
@@ -397,10 +397,13 @@ generated by the following algorithm:
 * `vdaf_setup() -> (public_param, verify_params: Vec[Bytes])` is the randomized
   setup algorithm used to generate the public parameter used by the Clients
   (`public_param`) and the verification parameters used by the Aggregators
-  (`verify_params`). The length of the latter MUST be `SHARES`. The parameters
-  are generated once and reused across multiple VDAF evaluations. In general, an
+  (`verify_params`). The length of the latter MUST be `SHARES`. In general, an
   Aggregator's verification parameter is considered secret and MUST NOT be
-  revealed to the Clients, Collector or other Aggregators.
+  revealed to the Clients, Collector or other Aggregators. The parameters are
+  generated once and MAY be reused across multiple VDAF evaluations. To avoid
+  leaking `verify_params` to Clients, it is RECOMMENDED that the Aggregators
+  generate them only after a set of Client inputs has been collected for
+  verification.
```

Comment:
I'd prefer this text to be in security considerations, along with a short explanation of the threat model being considered.

Reply:
Done.